README.md

```diff
@@ -1,59 +1,64 @@
 # ByteSize 
 
-`ByteSize` is a utility class that makes byte size representation in code easier by removing ambiguity of the value being represented.
+`ByteSize` is a utility class that makes byte size representation in code easier 
+by removing ambiguity of the value being represented.
 
 `ByteSize` is to bytes what `System.TimeSpan` is to time.
 
-[![](https://travis-ci.org/omar/ByteSize.svg?branch=master)](https://travis-ci.org/omar/ByteSize)
-[![Stable nuget](https://img.shields.io/nuget/v/ByteSize.svg)](https://www.nuget.org/packages/ByteSize/)
+[![](https://travis-ci.org/omar/DecimalByteSize.svg?branch=master)](https://travis-ci.org/omar/ByteSize)
+[![Stable nuget](https://img.shields.io/nuget/v/DecimalByteSize.svg)](https://www.nuget.org/packages/ByteSize/)
 
-#### Building
+#### Development
 
-* Windows: use Visual Studio
-* Mac OS X
-   * Install [Mono](http://www.mono-project.com/download/).
-     * NOTE: using `brew install mono` will not install the PCL libraries required to build the PCL compatible DLLs. The PCL libraries can be installed by running the installer downloaded from http://www.mono-project.com/download/. 
-   * Run `make build` in terminal.
-* Linux
-   * Install [Mono](http://www.mono-project.com/docs/getting-started/install/linux/) and the reference assemblies (`sudo apt-get referenceassemblies-pcl`).
-   * Run `make build` in terminal.
+- Install [.NET Core SDK](https://dotnet.microsoft.com/download)
+- Build: `make build`
+- Test: `make test`
 
-## Usage 
+## Usage
 
-`ByteSize` assumes `1 kilobyte` = `1024 bytes`. See [why here](http://omar.io/2017/01/16/when-technically-right-is-wrong-kilobytes.html).
+`ByteSize` comes with 3 different ways to represent bytes:
+
+- `DecimalByteSize` which assumes `1 kilobyte` = `1000 bytes` with 2 letter abbrevations `b`, `B`,`KB`, `MB`, `GB`, `TB`, `PB`.
```

Comment:
I've started to use your alpha-version and faced with problem with naming KB for Decimal. It seems that for Decimal it should be 'kB' instead of 'KB'.
https://en.wikipedia.org/wiki/Kilobyte 
https://en.wikipedia.org/wiki/Mega-

Reply:
It will be cool if you'll build version new alpha version before completing PR 😎